internals/secrethub/service_aws_init.go

```diff
@@ -84,12 +104,12 @@ func (cmd *ServiceAWSInitCommand) Run() error {
 // Register registers the command, arguments and flags on the provided Registerer.
 func (cmd *ServiceAWSInitCommand) Register(r Registerer) {
 	clause := r.Command("init", "Create a new AWS service account attached to a repository.")
-	clause.Flag("path", "The service account is attached to the repository in this path and when used together with --permission, an access rule is created on the directory in this path.").Required().SetValue(&cmd.path)
+	clause.Arg("repo", "The service account is attached to the repository in this path.").Required().SetValue(&cmd.repo)
 	clause.Flag("kms-key-id", "ID of the KMS-key to be used for encrypting the service's account key.").Required().StringVar(&cmd.kmsKeyID)
 	clause.Flag("role", "ARN of the IAM role that should have access to this service account.").Required().StringVar(&cmd.role)
 	clause.Flag("region", "The AWS region that should be used").StringVar(&cmd.region)
 	clause.Flag("desc", "A description for the service").StringVar(&cmd.description)
-	clause.Flag("permission", "Automatically create an access rule giving the service account permission on the given path argument. Accepts `read`, `write` or `admin`.").SetValue(&cmd.permission)
+	clause.Flag("permission", "Create an access rule giving the service account permission on a directory. Accepted permissions are `read`, `write` and `admin`. Use <subdirectory>:<permission> format to give permission on a subdirectory of the repo.").StringVar(&cmd.permission)
```

Comment:
Is the `<subdirectory>:`-part of this optional?

Reply:
Yes. If no subdirectory is supplied (e.g. `--permission read`), it is given on the root directory of the repo.

Reply:
Okay, then shouldn't we document that?

Reply:
I've updated the help-text. Does that help?

Reply:
Yep